### PR TITLE
docs: improve documentation to help user get started

### PR DESCRIPTION
**Pull request by jecfish**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,8 +23,17 @@ If you want to replay recordings using Puppeteer, install Puppeteer as well:
...
-## Replay a recording stored in a file using Puppeteer
+You can use Puppeteer Replay to:
+
+1. **Replay recording**. Replay recording with CLI or using [the replay lib API](/examples/replay-from-file-using-puppeteer/main.js).
+2. **Customize replay**. Customize how a recording is run (e.g. capture screenshots after each step, intergrate with 3rd party libraries).
+3. **Transform recoding**. Customize how a recording is stringified. (e.g. transform the recording into other format like [Cypress test script](https://github.com/cypress-io/cypress-chrome-recorder))
+
+## 1. Replay recording
+
+Download this [example recording](https://storage.googleapis.com/web-dev-uploads/file/dPDCek3EhZgLQPGtEG3y0fTn4v82/vzQbv2rUfTz2DEmx06Gv.json).
```
Comment (OrKoN) [inline README.md:36]: should we say to save it as `recording.js`?

Comment (jecfish) [inline README.md:36]:
```suggestion
Download this [example recording](https://storage.googleapis.com/web-dev-uploads/file/dPDCek3EhZgLQPGtEG3y0fTn4v82/vzQbv2rUfTz2DEmx06Gv.json) and save it as `recording.json`.
```

**Review by OrKoN — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 ###### [API](https://github.com/puppeteer/replay/blob/main/docs/api) | [Contributing](https://github.com/puppeteer/replay/blob/main/docs/contributing.md)
 
-> Replay is a library which provides an API to replay and stringify recordings created using [Chrome DevTools Recorder](https://developer.chrome.com/docs/devtools/recorder/)
+> Puppeteer Replay is a library which provides an API to replay and stringify recordings created using [Chrome DevTools Recorder](https://developer.chrome.com/docs/devtools/recorder/)
```
[the remark this answers is not in the document]
Comment (sofiayem) [reply README.md:11]: See https://www.dictionary.com/e/that-vs-which/

File: README.md [hunk 3]
```diff
@@ -23,8 +23,17 @@ If you want to replay recordings using Puppeteer, install Puppeteer as well:
 ```
 npm install puppeteer --save
 ```
+## Getting started with Puppeteer Replay
 
-## Replay a recording stored in a file using Puppeteer
+You can use Puppeteer Replay to:
+
+1. **Replay recording**. Replay recording with CLI or using [the replay lib API](/examples/replay-from-file-using-puppeteer/main.js).
+2. **Customize replay**. Customize how a recording is run (e.g. capture screenshots after each step, intergrate with 3rd party libraries).
```
Comment (sofiayem) [inline README.md:31]:
```suggestion
2. **Customize replay**. Customize how a recording is run. For example, capture screenshots after each step or integrate with 3rd party libraries.
```

File: README.md [hunk 4]
```diff
@@ -23,8 +23,17 @@ If you want to replay recordings using Puppeteer, install Puppeteer as well:
...
+3. **Transform recoding**. Customize how a recording is stringified. (e.g. transform the recording into other format like [Cypress test script](https://github.com/cypress-io/cypress-chrome-recorder))
```
Comment (sofiayem) [inline README.md:32]:
```suggestion
3. **Transform recoding**. Customize how a recording is stringified. For example, transform the recording into another format, like [Cypress test script](https://github.com/cypress-io/cypress-chrome-recorder))
```

File: README.md [hunk 5]
```diff
@@ -59,22 +68,12 @@ const runner = await createRunner(recording);
...
-console.log(await stringify({
-  title: 'Test recording',
-  steps: [],
-}));
-```
-
-## [Customize how a recording is run](/examples/extend-runner/main.js)
+## 2. Customize replay
 
 The library offers way to customize how a recording is run. You can extend
```
Comment (sofiayem) [inline README.md:73]:
```suggestion
The library offers a way to customize how a recording is run. You can extend
```

**Review by sofiayem — APPROVED** [review]

**Pull request merged**